rest-api-spec/src/main/resources/rest-api-spec/api/cat.indices.json

```diff
@@ -51,7 +51,11 @@
       },
       "local":{
         "type":"boolean",
-        "description":"Return local information, do not retrieve the state from master node (default: false)"
+        "description":"Return local information, do not retrieve the state from master node (default: false)",
+        "deprecated":{
+          "version":"8.0.0",
```

Comment:
Shouldn't this be the release in which we deprecate the parameter? That should be before 8.0.0.

Reply:
We typically update the version when backporting to a release branch and then come back and update the master branch.